recipes/ablang2/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "ablang2" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ablang2-{{ version }}.tar.gz
+  sha256: 7f01bec5c5a5d5c270fa078cdd923d3a98bdc2b93538b17d46231db2d783676d
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >3.6
```

```suggestion
    - python >=3.6
```
Or should it be 3.7?